docs/tutorials/compilation.rst

```diff
@@ -24,6 +24,10 @@ Optional but highly recommended
     ./configure --with-cxx --enable-shared --without-python --without-matlab --without-octave
     sudo make install
 
+.. caution::
+
+  The Debian/Unbuntu NLOpt package does NOT come with C++ bindings. Therefore you need to compile NLOpt yourself. The brew package (OSX) comes with C++ bindings (`brew install homebrew/science/nlop`).
```

Comment:
@jbmouret it's `brew install homebrew/science/nlop` or `brew install homebrew/science/nlopt`?